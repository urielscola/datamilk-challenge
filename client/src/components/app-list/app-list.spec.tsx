import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import AppList from './app-list';

describe('AppList component', () => {
  it('should render', () => {
    const result = render(
      <AppList data="downloads" apps={[]} label="Downloads" />
    );
    expect(result).toMatchSnapshot();
  });
});
