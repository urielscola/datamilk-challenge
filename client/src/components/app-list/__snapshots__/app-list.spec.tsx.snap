// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppList component should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styles__Container-sc-1pah7wz-0 ecSmOp"
        data-testid="widget"
      >
        <div
          class="styles__Container-sc-1rvgi7a-0 jvSBtk"
          data-testid="flex-div"
          display="flex"
        >
          <p
            class="sc-bdnxRM jhULRO"
            data-testid="text"
          >
            App name
          </p>
          <p
            class="sc-bdnxRM jhULRO"
            data-testid="text"
          >
            Downloads
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styles__Container-sc-1pah7wz-0 ecSmOp"
      data-testid="widget"
    >
      <div
        class="styles__Container-sc-1rvgi7a-0 jvSBtk"
        data-testid="flex-div"
        display="flex"
      >
        <p
          class="sc-bdnxRM jhULRO"
          data-testid="text"
        >
          App name
        </p>
        <p
          class="sc-bdnxRM jhULRO"
          data-testid="text"
        >
          Downloads
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
